### PR TITLE
delete fix & new delete message box

### PR DESCRIPTION
**Pull request by bthuntington**

### CODE REVIEW

File: yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html [hunk 1]
```diff
@@ -17,20 +17,46 @@
...
-                <button class="button" onclick="deleteFunctionWorkflows()">Delete Workflow</button>
+            <div class=delete>
+                <button onclick="deleteFunc()">Delete Workflow</button>
             </div>
             <script>
-                function deleteFunctionWorkflows() {
-                    if (confirm("Do you really want to delete this workflow?")) {
-                        document.location.href="{% url 'delete_workflows' workflow.id %}";
-                    }
+                function deleteFunc() {
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html]: I think you should call this deleteWorkflow(), it's obvious it's a function but without reading the code body it is unclear what is getting deleted.

File: yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html [hunk 2]
```diff
@@ -17,20 +17,46 @@
     {% comment %} HACK: For some reason this allows formatting to work, do not remove. {% endcomment %}
     <div class="filler"></div>
     <div class="border">
-        {% if edit == True %}
-        <div class="edit_button">
+     <div class=edit_button>
+        {% if edit == False %}
```
Comment (clundin55) [inline yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html]: Shouldn't this be true, and enclosing the edit button div? also i believe div class names should now have " " around the,

File: yw_website/yw_website/apps/website/models.py [hunk 3]
```diff
@@ -113,8 +113,8 @@ class Meta:
         unique_together = ("port_id", "run")
 
     port_id = models.IntegerField()
-    on_program_block = models.ForeignKey(ProgramBlock, on_delete=models.DO_NOTHING)
-    data = models.ForeignKey(Data, on_delete=models.DO_NOTHING)
+    on_program_block = models.ForeignKey(ProgramBlock, on_delete=models.CASCADE)
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/models.py:116]: nice catch

**Review by clundin55 — APPROVED** [review]
lgtm

**Pull request merged**